### ISSUE TITLE
Replace oidc-client with oidc-client-ts

### ISSUE THREAD

**Issue by hsandhar** [jasontaylordev/CleanArchitecture#825]
**Is your feature request related to a problem? Please describe.**
Currently used [oidc-client](https://github.com/IdentityModel/oidc-client-js) package is no longer maintained, and another successor project [oidc-client-ts](https://github.com/authts/oidc-client-ts) (forked from original package) is showing great promise. 

<img width="1009" alt="image" src="https://user-images.githubusercontent.com/1728547/224568962-6a489435-ebf4-4aea-a3f9-8f3b84eb77e0.png">
<img width="569" alt="image" src="https://user-images.githubusercontent.com/1728547/224568985-6ace999d-e267-453d-a4d0-1bd6f329ffd8.png">

**Describe the solution you'd like**
Replace [oidc-client](https://github.com/IdentityModel/oidc-client-js) with [oidc-client-ts](https://github.com/authts/oidc-client-ts) 

**Describe alternatives you've considered**
Why fix it if it's not broken, so yeah, sure, we could leave it as-is, but then again, why not make it better. Alternative would be to leave it as-is.

**Additional context**
For reference, I've created a [PR here](https://github.com/jasontaylordev/CleanArchitecture/pull/824) (left it in Draft intentionally), depending on the feedback, I could change the status to Ready for Review.

**To summarize**:
[This PR](https://github.com/jasontaylordev/CleanArchitecture/pull/824) more or less just replaces oidc-client with oidc-client-ts with some minor changes; more or less highlighted in the [migration guide](https://github.com/authts/oidc-client-ts/blob/main/docs/migration.md).

- Default of `loadUserInfo` changed from `true` → `false`
- `allowedCommonJsDependencies` within `angular.json`: replaced `oidc-client` (no longer used/older package) with `crypto-js` (one of the dependencies for oidc-client-ts. Just removes [build warnings](https://angular.io/guide/build#configuring-commonjs-dependencies).

**Comment by jasontaylordev** [comment]
Thanks for the contribution, however, I'm shifting the focus now to .NET 8, where the **oidc-client** is no longer in use.
